### PR TITLE
Provide option to prepend items to the showroom queue.

### PR DESCRIPTION
**Pull request by phgross**
Used for https://github.com/4teamwork/opengever.core/issues/1872.

@bierik  please have a 👀

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -315,6 +315,26 @@ or
 
 ------------
 
+**Showroom.prepend**
+
+Extend the current item queue with new items, similar than the ``append`` method, but the items are prepended at the beginning of the queue.
```
Comment (deiferni) [inline README.rst]: similar than -> similar to

File: ftw/showroom/js/test/spec/showroom.js [hunk 2]
```diff
@@ -460,6 +460,58 @@ describe("Showroom", () => {
 
   });
 
+
+  describe("prepend", () => {
+
+    it("should extend the current item set on beginning", () => {
```
Comment (deiferni) [inline ftw/showroom/js/test/spec/showroom.js]: set on beginning -> list at the top

**Comment by deiferni** [comment]
Looks 👍

**Comment by phgross** [comment]
@deiferni updated ...

**Comment by bierik** [comment]
@phgross 👍  thx 💛

**Pull request merged**